File: 8_kyu/Grasshopper_Grade_book.php
<?php

function getGrade($a, $b, $c) {
    $media = ($a + $b + $c) / 3;

    if ($media < 60) {
        return 'F';
    } else if ($media < 70) {
        return 'D';
    } else if ($media < 80) {
        return 'C';
    } else if ($media < 90) {
        return 'B';
    } else if ($media <= 100) {
        return 'A';
    }
}

File: 8_kyu/Grasshopper_Grade_book.java
public class GrassHopper {

    public static char getGrade(int s1, int s2, int s3) {
        int media = (s1 + s2 + s3) / 3;

        if (media < 60) {
            return 'F';
        } else if (media < 70) {
            return 'D';
        } else if (media < 80) {
            return 'C';
        } else if (media < 90) {
            return 'B';
        } else {
            return 'A';
        }

    }
}

